tools/tune/eq/example_iir_bandsplit.m

```diff
@@ -32,7 +32,7 @@ function example_iir_bandsplit()
 
 %% Build blob
 channels_in_config = 4;      % Setup max 4 channels EQ
-assign_response = [0 1 0 1]; % Order: lo, hi, lo, hi
+assign_response = [0 0 1 1]; % Order: lo, lo, hi, hi
```

Comment:
and that too?

Reply:
here we change the eq to lo-lo-hi-hi since we want odm to use the channel order "L_lo-R_lo-L_hi-R_hi". of cause they can overwrite the order by defining their own demux routing matrix in the topology and using their own eq coefficient.

Reply:
yes but those files are examples and not meant to represent the channel order of each and every ODM.

Reply:
The coefficient generated by this script will be included in pipe-demux-eq-iir-playback.m4 as default EQ value so I modify this file and also the pipe-demux-eq-iir-playback.m4.

Reply:
Yep, OK to modify. If there are later other band-split choices they can be added to script. My example was just a guess of channels assign.